main.js

```diff
@@ -16,7 +18,8 @@ const websocketServer = require(source);
  * Initialize the websocket server
  */
 
-const WebSocket = require('ws');
-const wss = new WebSocket.Server({ port: process.env.PORT || 5000 });
+const server = http.createServer();
```

Comment:
Seems a little weird to create a full-blown http server for the nasonov writer version as well, but I agree that it's probably more elegant to do it here.